cypress/fixtures/navigationCategories.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
This is not a json, this is just an array, no need to have a whole file for this, you can add this array into another json or if this json will be bigger you can add this array into a json inside this file

Reply:
okay, i just separated them based on what they were used for